opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/_view_instrument_match.py

```diff
@@ -74,8 +74,8 @@ def conflicts(self, other: "_ViewInstrumentMatch") -> bool:
                 result
```

Comment:
Changes to this file are just renaming of variables, not much review is necessary here.